src/AppNavigator.tsx

```diff
@@ -149,14 +147,16 @@ const CreatePinScreenWrapper = () => {
   return <EnableLoginWithPin onDone={login} />
 }
 
-const StoryBook = () => (
-  <Stack.Navigator
-    screenOptions={{
-      detachPreviousScreen: false /* https://github.com/react-navigation/react-navigation/issues/9883 */,
-    }}
-  >
-    <Stack.Screen name="storybook" component={StorybookScreen} />
-  </Stack.Navigator>
+export const StoryBookNavigator = () => (
```

Comment:
get rid of this, and just use StorybookScreen in the stack navigator

Reply:
`<Stack.Screen name="storybook" component={StorybookScreen} options={{headerShown: true}}/>`